LoopKitUI/Views/Settings Editors/TherapySettingsView.swift

```diff
@@ -229,7 +230,56 @@ extension TherapySettingsView {
             }
         }
     }
+        
+    var insulinModelSection: some View {
```

Comment:
Why does the therapy settings view show all of the available insulin models? I would expect only the selected one to be visible in the overview.

Reply:
This exact question was already raised [in the Figma file](https://www.figma.com/file/7da2yRPMUEdQnaQq3KvgZq?node-id=85:11939#33498184).  I interpreted the design as showing all of them.  It would not be hard to change.